tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
```

Comment:
Google-style requires `#!/bin/bash`, because this removes a vulnerability vector. I know that Macs have issues with hideously old bash for which you need homebrew replacements, but you shouldn't need any of the bash>3.0 features, anyway.